docs/docs/ionic-angular/installation.md

```diff
@@ -0,0 +1,91 @@
+# Getting Started
+
+## Install Plugin
+
+This plugin does need the angular and capacitor plugin, so add them to your Nx workspace:
+
+```bash
+npm install @nxext/capacitor --save-dev
+npm install @nx/angular --save-dev
```

Comment:
@edbzn Tbh, I weren't aware that I had to download `@nx/angular` before being able to use the `@nxext/angular-ionic` so it could still be helpful.

Or do we wish to make `@nx/angular` being a dependency of the `@nxext/angular-ionic` library ? (It could be useful)

Reply:
> @edbzn Tbh, I weren't aware that I had to download @nx/angular before being able to use the @nxext/angular-ionic so it could still be helpful.

I'd prefer to emphasize that generating an Angular application is the first step and guide the user to the Nx documentation to do so, but it's not necessary to document how to do it.

We could add a note like this: 

> [!NOTE]  
> This plugin configures an **already existing Angular application**. Please, visit the [Nx documentation](https://nx.dev/nx-api/angular) to generate an application before continuing.

> Or do we wish to make @nx/angular being a dependency of the @nxext/angular-ionic library ? (It could be useful)

No, that could cause issues, and it is outside of our scope.